core/src/main/scala/implicits.scala

```diff
@@ -18,9 +18,9 @@ package freestyle.cassandra
 
 import cats.{~>, MonadError}
 import com.datastax.driver.core.{Cluster, Session}
-import freestyle.async.AsyncContext
-import freestyle.async.{Implicits => AsyncImplicits}
-import freestyle.asyncGuava.AsyncGuavaImplicits
+import freestyle.free.async.AsyncContext
```

Comment:
I think the AsyncContext is generic enough that it's like in RPC and Cassandra libs where it does not need a free and a tagless version.